src/app/App.js

```diff
@@ -1,6 +1,9 @@
 import Box from './components/Box';
 import Button from './components/Button';
+import { RankingBox } from './components/RankingBox';
 export const App = ({ options }) => {
   new Box('MODE: who is this character', 'box');
   new Button('button', 'play the game');
+  const scores = [{player: "Anna", correctAnswers: 10, allAnswers:20},{player: "Kamil", correctAnswers: 7, allAnswers:20}, {player: "Ela", correctAnswers: 3, allAnswers:20}];
```

Comment:
@KlaraGajaszek @maciekkwiet tutaj dodałam zmienną z przykładową tablicą wyników, żeby móc stworzyć obiekt klasy `RankingBox` , zostawić ją czy usunąć?

Reply:
Jest okey, niech zostanie, później to zmienimy :D